### ISSUE TITLE
Doesn't work as expected on safari 12.0 

### ISSUE THREAD

**Issue by eoinpaul** [Simonwep/pickr#114]
#### Do you want to request a **feature** or report a **bug**? 
Report a bug

#### What is the current behavior?
When I use Pickr in safari all of the swatches at the bottom are black and the color doesn't change when I click on the save button.

#### If the current behavior is a bug, please provide the steps to reproduce
Go to the demo page (https://simonwep.github.io/pickr/) in safari 12.0 
All the swatches are black and the color doesn't change when I click save

#### What is the expected behavior?
Swatches should be different colors and should change when I click save.

#### Your environment:
```
Version (see Pickr.version): latest
Used bundle (es5 or normal one): es5
Used theme (default is classic): all themes
Browser-version:  safari 12.0
Operating-system:  macOS Mojave 10.14
```

**Comment by eoinpaul** [comment]
<img width="746" alt="Screenshot 2019-06-28 at 12 49 39" src="https://user-images.githubusercontent.com/5315435/60340343-376b2b00-99a3-11e9-8050-d68f98be8e8c.png">

**Comment by Simonwep** [comment]
Nice, safari seems to be not able to round / accept floating point `rgba` values.
Fixed in 63d45c0582ba8f9399cf8607cd3fd40b348c6a26, thanks for the report :)
